templates/zerver/integrations.html

```diff
@@ -1801,6 +1801,54 @@
 
     </div>
 
+    <div id="papertrail" class="integration-instructions">
+
+
+      <p>
+        Zulip supports integration with Papertrail as a
+        <a href="http://www.perforce.com/perforce/doc.current/manuals/p4sag/chapter.scripting.html">Webhook</a>
+        that fires upon finding a certain log. To do this:
+      </p>
+
+      <p>
+        First, create the stream you'd like to use for Papertrail
+        notifications, and subscribe all interested parties to this stream. We
+        recommend the stream name be <code>papertrail</code>. After create a bot named
+        <code>Papertrail</code> and note the API key.
+      </p>
+
+      <p>
+        Next, log into your Papertrail Account and browse through your logs.
+        Search for logs you want to get alerts for and press the <code>Save Search</code>,
+        though be sure to press the <code>Save & Setup an alert button</code>
+      </p>
+
+      <img class="screenshot" src="/static/images/integrations/papertrail/000.png" />
+      <img class="screenshot" src="/static/images/integrations/papertrail/001.png" />
```

Comment:
Add some space or text between these two screenshots. Currently, it looks like that there is just one image.
![screen shot 2017-01-11 at 10 00 27 am](https://cloud.githubusercontent.com/assets/641763/21836633/79104718-d7e8-11e6-99c3-dff0d4a1432c.png)